packaging/Dockerfile.armv7

```diff
@@ -1,10 +1,10 @@
-FROM --platform=linux/amd64 alpine:3.13 AS build
+FROM --platform=linux/arm/v7 alpine:3.13
```

Comment:
`--platform` is not strictly required but it ensures that we use the correct architecture for the added `arm` binaries below.